tests/integration/modules/mac_timezone.py

```diff
@@ -0,0 +1,175 @@
+
+        USE_NETWORK_TIME = self.run_function('timezone.get_using_network_time')
+        TIME_SERVER = self.run_function('timezone.get_time_server')
+        TIME_ZONE = self.run_function('timezone.get_zone')
+        CURRENT_DATE = self.run_function('timezone.get_date')
+        CURRENT_TIME = self.run_function('timezone.get_time')
+
+        self.run_function('timezone.set_using_network_time', [False])
+
+    def tearDown(self):
+        '''
+        Reset to original settings
+        '''
+        self.run_function('timezone.set_time_server', [TIME_SERVER])
+        self.run_function('timezone.set_using_network_time', [USE_NETWORK_TIME])
+        self.run_function('timezone.set_zone', [TIME_ZONE])
+        if not USE_NETWORK_TIME:
+            self.run_function('timezone.set_date', [CURRENT_DATE])
+            self.run_function('timezone.set_time', [CURRENT_TIME])
+
+    def test_get_set_date(self):
+        '''
+        Test timezone.get_date
+        Test timezone.set_date
+        '''
+        # Correct Functionality
+        self.assertTrue(self.run_function('timezone.set_date', ['2/20/2011']))
+        self.assertEqual(self.run_function('timezone.get_date'), '2/20/2011')
+
+        # Test bad date format
+        self.assertEqual(
+            self.run_function('timezone.set_date', ['13/12/2014']),
+            'ERROR executing \'timezone.set_date\': '
+            'Invalid Date/Time Format: 13/12/2014'
+        )
+
+    def test_get_set_time(self):
+        '''
+        Test timezone.get_time
+        Test timezone.set_time
+        '''
+        # Correct Functionality
+        self.assertTrue(self.run_function('timezone.set_time', ['3:14']))
+        new_time = self.run_function('timezone.get_time')
+        new_time = datetime.strptime(new_time, '%H:%M:%S').strftime('%H:%M')
+        self.assertEqual(new_time, '03:14')
+
+        # Test bad time format
+        self.assertEqual(
+            self.run_function('timezone.set_time', ['3:71']),
+            'ERROR executing \'timezone.set_time\': '
+            'Invalid Date/Time Format: 3:71'
+        )
+
+    def test_get_set_zone(self):
+        '''
+        Test timezone.get_zone
+        Test timezone.set_zone
+        '''
+        # Correct Functionality
+        self.assertTrue(self.run_function('timezone.set_zone',
+                                          ['Pacific/Wake']))
+        self.assertEqual(self.run_function('timezone.get_zone'), 'Pacific/Wake')
+
+        # Test bad time zone
+        self.assertEqual(
+            self.run_function('timezone.set_zone', ['spongebob']),
+            'ERROR executing \'timezone.set_zone\': '
+            'Invalid Timezone: spongebob')
+
+    def test_get_offset(self):
+        '''
+        Test timezone.get_offset
+        '''
+        self.assertTrue(self.run_function('timezone.set_zone',
+                                          ['Pacific/Wake']))
+        self.assertEqual(self.run_function('timezone.get_offset'), '+1200')
```

Comment:
Added another set where it changes to 'America/Denver' and then checks the offset.